### ISSUE TITLE
How do you use the 'either' type?

### ISSUE THREAD

**Issue by mlockrey** [joegesualdo/get-youtube-subtitles-node#2]
I've had no trouble getting the subtitles - one at a time (i.e. Published English track = 'nonauto' and the automatic craptions = 'auto').

But how do I use the 'either' type ? 

It doesn't seem to deliver anything ?

Cheers, Mike L

**Comment by hhkaos** [comment]
Does it works for you?, I'm trying to download [this video](https://www.youtube.com/watch?v=uHDkhm3QiTY&list=PLaPDDLTCmy4aJoXyro9yWOr2pdIe0mr9A) auto-subtitles this:

```js
var getYoutubeSubtitles = require('@joegesualdo/get-youtube-subtitles-node');

getYoutubeSubtitles('uHDkhm3QiTY', {type: 'auto'})
.then(subtitles => {
  console.log(subtitles)
})
.catch(err => {
  console.log(err)
})
```
And console returns: `Auto Subtitles dont exist for uHDkhm3QiTY`

Any ideas?

----

@mlockrey if you check [@joegesualdo's code](https://github.com/joegesualdo/get-youtube-subtitles-node/blob/master/index.js#L7), it seems that **either** parameters is not an option in this script. It is only valid for [this package](https://github.com/joegesualdo/get-youtube-subtitle-url-node/blob/master/index.js#L67).

Cheers!

**Comment by joegesualdo** [comment]
@mlockrey Looks like I forgot to add the `either` case, but you can just check for `auto` and `nonauto` and mimic the desired `either` functionality.

@mlockrey @hhkaos Looks like there was a broken dependency which was breaking this package. I've fixed the issue and updated to version `0.1.0`.

I'd update and try again.

**Comment by joegesualdo** [comment]
Closing for now. Let me know if it didn't solve your issue or you have any other problems.
